bin/sumd

```diff
@@ -128,18 +131,22 @@ def load_configs(schedule):
         schedule.add(task)
 
 def riemann_client():
-    if options.client_key_file != None and options.client_cert_file != None:
-        riemann = bernhard.SSLClient(
-            host=options.riemann_host,
-            port=options.riemann_port,
-            keyfile=options.client_key_file,
-            certfile=options.client_cert_file,
-            ca_certs=options.ca_certs_file)
-    else:
-        riemann = bernhard.Client(
-            host=options.riemann_host,
-            port=options.riemann_port,
-            transport=bernhard.UDPTransport)
+    riemann = []
+    if type(options.riemann_host) not in (list, tuple):
+            options.riemann_host = [options.riemann_host]
+    for h in options.riemann_host:
+        if options.client_key_file != None and options.client_cert_file != None:
```

Comment:
This option is generally host-specific, and would require the Riemann servers to be configured very similarly (from a client ssl perspective) to work, because this is still a single option in the configuration.

Perhaps I could help suggest some improvements here, but I don't think I fully understand the use case yet. Can you expand on your architecture and goal here?